packages/components/src/theme/assays.scss

```diff
@@ -2,3 +2,11 @@
 .import-rename-filename {
   color: $brand-primary
 }
+
+.gray-text {
+    color: $gray-light;
+}
+
+.small-margin-bottom {
+    margin-bottom: 5px;
+}
```

Comment:
Very minor but these seem like general rules that could be used outside of assays, thoughts on moving them to a more general scss file?